src/bindings/js/node/src/helper.cpp

```diff
@@ -136,45 +107,35 @@ std::unordered_set<std::string> js_to_cpp<std::unordered_set<std::string>>(
 }
 
 template <>
-ov::element::Type_t js_to_cpp<ov::element::Type_t>(const Napi::CallbackInfo& info,
-                                                   const size_t idx,
-                                                   const std::vector<napi_types>& acceptable_types) {
+ov::element::Type_t js_to_cpp<ov::element::Type_t>(const Napi::CallbackInfo& info, const size_t idx) {
     const auto elem = info[idx];
-    if (!acceptableType(elem, acceptable_types))
-        OPENVINO_THROW(std::string("Cannot convert Napi::Value to ov::element::Type_t"));
+    OPENVINO_ASSERT(elem.IsString(), "Cannot convert Napi::Value to ov::element::Type_t");
```

Comment:
What is about more specific message?
```suggestion
    OPENVINO_ASSERT(elem.IsString(), "Passed argument should be a String type");
```